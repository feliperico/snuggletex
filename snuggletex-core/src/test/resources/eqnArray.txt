\begin{eqnarray}
x' &=& &x \sin\phi &+& z \cos\phi \\
z' &=& - &x \cos\phi &+& z \sin\phi \\
\end{eqnarray}