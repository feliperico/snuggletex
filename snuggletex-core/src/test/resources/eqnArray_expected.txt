<math xmlns="http://www.w3.org/1998/Math/MathML" display="block">
  <mtable>
    <mtr>
      <mtd form="infix" lspace="mediummathspace">
        <msup>
          <mi>x</mi>
          <mi>'</mi>
        </msup>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mo>=</mo>
      </mtd>
      <mtd form="infix" lspace="mediummathspace"/>
      <mtd form="infix" lspace="mediummathspace">
        <mi>x</mi>
        <mi>sin</mi>
        <mi>φ</mi>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mo>+</mo>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mi>z</mi>
        <mi>cos</mi>
        <mi>φ</mi>
      </mtd>
    </mtr>
    <mtr>
      <mtd form="infix" lspace="mediummathspace">
        <msup>
          <mi>z</mi>
          <mi>'</mi>
        </msup>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mo>=</mo>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mo>-</mo>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mi>x</mi>
        <mi>cos</mi>
        <mi>φ</mi>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mo>+</mo>
      </mtd>
      <mtd form="infix" lspace="mediummathspace">
        <mi>z</mi>
        <mi>sin</mi>
        <mi>φ</mi>
      </mtd>
    </mtr>
  </mtable>
</math>